### PR TITLE
i18n(es): Add preprcms.mdx

### PR DESCRIPTION
**Pull request by AitorMT**
#### Description (required)

Spanish `preprcms.mdx` has been added.

### CODE REVIEW

**Review by dreyfus92 — COMMENTED** [review]
Hey @AitorMT, thank you for handling this translation but unfortunately we cannot translate this one yet since the prop `i18nReady` is set to false. We might need to put on hold this one for sometime until it is set to true and it is ready for being translated. cc @Yan-Thomas

**Comment by AitorMT** [comment]
Okay, no problem, thanks for letting me know. Then I close the pull request, right ?

**Comment by dreyfus92** [comment]
Let's wait for Yan to see what he suggests. Maybe we can just put it on hold for later, what I mean is just to leave the PR there so when the appropriate time comes we can merge it.

**Comment by Yan-Thomas** [comment]
We have an issue to check these pages to see if they're good to be `i18nReady: true`, so let's hold on for now! No need to close 🙌

**Review by Waxer59 — COMMENTED** [review]
Buena PR 🤠, te dejo unas sugerencias!

File: src/content/docs/es/guides/cms/preprcms.mdx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: Prepr CMS & Astro
+description: Añada contenido a su proyecto Astro utilizando Prepr como CMS
+type: cms
+service: Prepr CMS
+stub: true
+i18nReady: false
```
Comment (Waxer59) [inline src/content/docs/es/guides/cms/preprcms.mdx:7]:
```suggestion
i18nReady: true
```

**Review by dreyfus92 — APPROVED** [review]
Thanks peeps, LGTM ✅

**Pull request merged**